docs/design/ovs-pipeline.md

```diff
@@ -314,34 +326,48 @@ more information on connection tracking in OVS.
 This table handles all "tracked" packets (all packets are moved to the tracked
 state by the previous table, [ConntrackTable]). It serves the following
 purposes:
- * keeps track of connections going through the gateway port; for all reply
- packets belonging to such connections we overwrite the destination MAC to the
- local gateway MAC on their way to the gateway port. This is required to handle
- the case were a Pod connects to a service and traffic goes through DNAT. In
- this case, if we do not use connection tracking, reply traffic from the backend
- will go directly to the originating Pod without going first through the gateway
- and kube-proxy. This means that the reply traffic will arrive at the
- originating Pod with the incorrect source IP (it will be set to the backend's
- IP instead of the service IP). So we keep track of such connections
- (identified by matching the ingress port against the gateway port) and for all
- reply traffic we set the destination to the local gateway MAC.
+
+ * keeps track of connections initiated through the gateway port, i.e. for which
+ the first packet of the connection (SYN packet for TCP) was received through
+ the gateway. For all reply packets belonging to such connections we overwrite
+ the destination MAC to the local gateway MAC to ensure that they get forwarded
+ though the gateway port. This is required to handle the following cases:
+   - reply traffic for Pod-to-ClusterIP traffic, which is handled by kube-proxy
+     and goes through DNAT. In this case the destination IP address of the reply
```

Comment:
Probably mention the client is a local Pod?

Reply:
Done